scripts/postgres-wait

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Before running migrations against a containerised postgres in local dev, this script helps make sure the container is started first.